preprocessing-pipeline-family.yaml

```diff
@@ -1,2 +0,0 @@
-name: general
```

Comment:
looks like this file is used in docker build ci: https://github.com/Unstructured-IO/unstructured-api/blob/main/.github/workflows/docker-publish.yml#L140
also i feel like we could still keep this file?